server/statistics_handler.go

```diff
@@ -128,21 +131,30 @@ func (sh StatsHistoryHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
 		return
 	}
 	snapshot := oracle.GoTimeToTS(t1)
-	is, err := sh.do.GetSnapshotInfoSchema(snapshot)
+	tbl, err := getSnapshotTableInfo(sh.do, snapshot, params[pDBName], params[pTableName])
 	if err != nil {
-		writeError(w, err)
-		return
+		logutil.BgLogger().Info("fail to get snapshot TableInfo in historical stats API, switch to use latest infoschema", zap.Error(err))
```

Comment:
better to clarify this in the doc？

Reply:
Currently, the historical stats lack docs. This PR aims to make historical stats at least usable as soon as possible since it's already been enabled now.
The behavior changed here is a bit too detailed for the user. I think we can clarify this when we add doc for the historical stats.